### ISSUE TITLE
Reload Website Data Fails

### ISSUE THREAD

**Issue by mam10eks** [tira-io/tira#171]
In the tira-admin interface, I click on Reload Website Data and nothing happens.
In the debug tools, I can see that the request gets an 500 response "{"failed":"FAILED"}".

**Comment by mam10eks** [comment]
It looks like this is the error in the frontend-pod:

```
Arguments: (AttributeError("module 'tira.tira_model' has no attribute 'build_model'"),)
--- Logging error ---
Traceback (most recent call last):
  File "./tira/endpoints/admin_api.py", line 21, in admin_reload_data
    model.build_model()
AttributeError: module 'tira.tira_model' has no attribute 'build_model'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1081, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 925, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 664, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 369, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/usr/lib/python3.8/site-packages/django/core/handlers/wsgi.py", line 133, in __call__
    response = self.get_response(request)
  File "/usr/lib/python3.8/site-packages/django/core/handlers/base.py", line 130, in get_response
    response = self._middleware_chain(request)
  File "/usr/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/usr/lib/python3.8/site-packages/django/utils/deprecation.py", line 126, in __call__
    response = response or self.get_response(request)
  File "/usr/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/usr/lib/python3.8/site-packages/django/utils/deprecation.py", line 126, in __call__
    response = response or self.get_response(request)
  File "/usr/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/usr/lib/python3.8/site-packages/django/utils/deprecation.py", line 126, in __call__
    response = response or self.get_response(request)
  File "/usr/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/usr/lib/python3.8/site-packages/django/utils/deprecation.py", line 126, in __call__
    response = response or self.get_response(request)
  File "/usr/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/usr/lib/python3.8/site-packages/django/utils/deprecation.py", line 126, in __call__
    response = response or self.get_response(request)
  File "/usr/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/usr/lib/python3.8/site-packages/django/utils/deprecation.py", line 126, in __call__
    response = response or self.get_response(request)
  File "/usr/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/usr/lib/python3.8/site-packages/django/utils/deprecation.py", line 126, in __call__
    response = response or self.get_response(request)
  File "/usr/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/usr/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./tira/checks.py", line 38, in func_wrapper
    return func(request, *args, **kwargs)
  File "./tira/endpoints/admin_api.py", line 26, in admin_reload_data
    logger.exception(f"/admin/reload_data failed with {e}", e)
Message: "/admin/reload_data failed with module 'tira.tira_model' has no attribute 'build_model'"
Arguments: (AttributeError("module 'tira.tira_model' has no attribute 'build_model'"),)
```

**Comment by BastianGrahm** [comment]
The function `tira_model.build_model()` is called but does not exist.
`build_model` is a function in the `_FileDatabase` object. However an instance of the `_FileDatabase` object exists in `tira_model.py` named `model`. So either `model.model.build_model() should be called or a global `build_model` function should be implemented.

**Comment by MattiWe** [comment]
Ah I see. 
Previously, the FileDatabase Module kept an instance of the FileDatabase object and all other Modules just imported the module. 
I changed this in an effort to fix #166. The imported `model` is now the Module, not the object, and the Module has all public functions. Since build_model() is a method of the object and not a function of the module, the call fails. The fix would be: make a Module-level function `build_model()` that calls the object function. 

That wont fix the issue that build_model is not working due to the multiprocessing :D

**Comment by BastianGrahm** [comment]
Sorry. My weeks are a mess right now.
I think its a little over to open a new branch for this so I fixed it in the dataset-revision-page branch.
A pull request will follow later today when I finished #169.
